pytest-embedded-wokwi/pytest_embedded_wokwi/arduino.py

```diff
@@ -12,4 +12,4 @@ class ArduinoFirmwareResolver:
 
     def resolve_firmware(self, app: 'ArduinoApp'):
         # get path of ino.bin file
-        return Path(app.binary_path, app.sketch + '.ino.bin')
+        return Path(app.binary_path, app.sketch + '.ino.merged.bin')
```

Comment:
One question, is this file auto generated by arduino? if no we can't do the rename here I suppose...

besides, the naming in `wokwi_cli.py` looks a bit weird...

```
        flasher_args = self.firmware_resolver.resolve_firmware(app) # <-- should be firemware?
        wokwi_toml_path = os.path.join(app.app_path, 'wokwi.toml')
        firmware_path = Path(os.path.relpath(flasher_args, app.app_path)).as_posix()
        elf_path = Path(os.path.relpath(app.elf_file, app.app_path)).as_posix()
```

could you help update it as well?

Reply:
Yes, this file is auto generated. So we can change use it consistently.

About the naming I can take a look but probably @urish knows if it is correct or not.

Reply:
no problem. thanks.